keyboards/neson_design/nico/info.json

```diff
@@ -0,0 +1,126 @@
+{
+    "manufacturer": "Neson Design",
+    "keyboard_name": "Nico",
+    "maintainer": "yulei",
+    "bootloader": "atmel-dfu",
+    "bootmagic": {
+        "enabled": true
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
+        "backlight": false,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "indicators": {
+        "caps_lock": "E2"
+    },
+    "matrix_pins": {
+        "cols": ["C7", "C6", "B6", "B5", "B4", "D7", "D6", "D4", "D5", "D3", "D2", "B7", "B3", "B2", "B1", "F6", "F7"],
+        "rows": ["F0", "E6", "F5", "F4", "F1"]
+    },
+    "processor": "atmega32u4",
+    "rgblight": {
+        "brightness_steps": 8,
+        "hue_steps": 8,
+        "led_count": 5,
+        "pin": "B0",
+        "saturation_steps": 8
+    },
+    "url": "",
+    "usb": {
+        "device_version": "0.0.1",
+        "no_startup_check": true,
```

Comment:
Is there a particular reason for this?  Namely, it's rather unusual to have this enabled at the kb level.

Reply:
The keyboard may unresponse when plugged to some USB HUB(bus powered) during suspend/resume, enable this can avoid this issue.  It was not only happened on this PCB, I have tested many others (not only those made by myself). It should be compatibility issue caused by the cheap USB HUB.